### PR TITLE
Get Travis-CI build going

### PR DESCRIPTION
**Pull request by hnrkndrssn**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+language: CSharp
+install: 
+ - echo "deb http://download.mono-project.com/repo/debian nightly main" | sudo tee /etc/apt/sources.list.d/mono-nightly.list
```
Comment (davidfowl) [inline .travis.yml]: There's a more first class way to do that

Comment (hnrkndrssn) [inline .travis.yml]: Do you mean `sudo add-apt-repository`

**Comment by hnrkndrssn** [comment]
https://github.com/travis-ci/travis-build/pull/460

Waiting for the above PR to be merged, then we can use:

```
mono:
 - nightly
```

to install the new Mono weekly snapshot builds which have fixes for known issues in Mono causing ASP.NET 5 builds to fail.

**Comment by hnrkndrssn** [comment]
Latest stable mono with fixes for the issues previously experienced has been released, so no need to specify `nightly` channel, `latest` works fine now.

**Comment by davidfowl** [comment]
Why KoreBuild?

**Comment by hnrkndrssn** [comment]
All the other dnx projects I looked at used it...

Wanna use this instead?

https://github.com/JabbR/JabbRv2/blob/dev/build.sh

**Comment by davidfowl** [comment]
Yep. Low tech :smile:

**Comment by hnrkndrssn** [comment]
:+1:

**Comment by hnrkndrssn** [comment]
Low tech build scripts implemented :bowtie:

**Comment by davidfowl** [comment]
:shipit:

**Comment by hnrkndrssn** [comment]
Ready for TravisCI to be enabled on JabbRv2 (which I can't do as I don't have admin access) :grimacing:

**Pull request merged**